internal/mode/static/nginx/config/maps_test.go

```diff
@@ -88,7 +88,10 @@ func TestExecuteMaps(t *testing.T) {
 		"map $http_upgrade $connection_upgrade {":                             1,
 	}
 
-	maps := string(executeMaps(conf))
+	mapResult := executeMaps(conf)
+	maps := string(mapResult[0].data)
```

Comment:
This should be called after `g.Expect(mapResult).To(HaveLen(1))` to prevent a possible panic if there are no entries. Same goes for all other tests like this.

Reply:
weird, added these changes before when @pleshakov mentioned.  See these changes locally present too

Reply:
Well not seeing them in the PR. Maybe commit wasn't pushed? Just need to check the length first before indexing into the arrays.

Reply:
oh my bad ! i misunderstood, updating